pkgs/by-name/vo/vo1ded-panel/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  ags,
+  astal,
+  dart-sass,
+  fetchFromGitHub,
+  fzf,
+  gjs,
+  gnused,
+  gobject-introspection,
+  gtk3,
+  gtk4-layer-shell,
+  stdenvNoCC,
+  wrapGAppsHook3,
+  wrapGAppsHook4,
+}:
```

Comment:
If you could clean up the input now that you aren't importing bundle directly that would be great.